frontend/src/components/LandingPage/Team/Team.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {
+  Card, CardTitle, CardText, CardGroup, CardBody
+} from 'reactstrap';
+
+const Team = (props) => {
+  return (
+    <div>
+      <h1> Meet the Team Members</h1>
+      <CardGroup>
+        <Card>
+          <CardBody>
+            <CardTitle>Vishnu Bapanapalli</CardTitle>
+            <CardText align='left'>Computer Science
+            <br/>
+            <a href='https://github.com/vishnubapana'>github</a>
+            <br/>
+            <CardText align='left'>Graduated with a Bachelors in Computer Engineering and 
+                currently pursuing a Masters in Computer Science with a focus in 
+                Intelligent Systems and Data Science.</CardText>
```

Comment:
React is giving a warning here about having a p tag be a [descendant of another p tag](https://imgur.com/a/rZ19ipv). The way to fix this is to have all of this text exist in one CardText. The same change would need to happen to the other cards as well.

Reply:
Updated the code to have all text in one CardText.